test/libsolidity/SemanticTest.cpp

```diff
@@ -508,17 +508,27 @@ TestCase::TestResult SemanticTest::runTest(
 			_stream << errorReporter.format(_linePrefix, _formatted);
 		}
 		_stream << endl;
+
 		AnsiColorized(_stream, _formatted, {BOLD, CYAN}) << _linePrefix << "Obtained result:" << endl;
+
 		for (TestFunctionCall const& test: m_tests)
 		{
 			ErrorReporter errorReporter;
-			_stream << test.format(
+			string outputString =
+			test.format(
 				errorReporter,
 				_linePrefix,
 				m_gasCostFailure ? TestFunctionCall::RenderMode::ExpectedValuesActualGas : TestFunctionCall::RenderMode::ActualValuesExpectedGas,
 				_formatted,
-				/* _interactivePrint */ true
-			) << endl;
+				true
```

Comment:
Why remove the comment?

Reply:
I think the comment got deleted when I was messing with the method inputs. There is no reason I can think of to remove it so I will put it back in.